Doc/library/typing.rst

```diff
@@ -32,6 +32,11 @@ In the function ``greeting``, the argument ``name`` is expected to be of type
 :class:`str` and the return type :class:`str`. Subtypes are accepted as
 arguments.
 
+New features have been added to the typing module in each major version of
```

Comment:
`have been` -> `are`?

(not a native speaker)

Reply:
I think "have been" is right. "are" would imply that we keep adding new features to each version, which has been true so far but isn't something we can promise will remain true :)

But separately, I think "major version" is wrong. Python 3 is a major version, but here we're talking about 3.9, 3.10, etc. Those are minor versions.
```suggestion
New features have been added to the typing module in each version of
```